dev/faq/tidb.md

```diff
@@ -598,6 +598,13 @@ Now, there are still a few reasons for this error reporting (the latter two are
 > + For each DDL operation, the number of `schema` version changes is the same with the number of corresponding `schema state` version changes.
 > + Different DDL operations cause different number of `schema` version changes. For example, the `CREATE TABLE` statement causes one `schema` version change while the `ADD COLUMN` statement causes four.
 
+#### What are the causes of the "Information schema is out of date" error
+
+When executing a DML statement, if TiDB fails to load the latest schema within a DDL lease (45s by default), the `Information schema is out of date` error might occur. Possible causes are:
+
+- The TiDB instance that executed this DML was killed, and the transaction execution corresponding to this DML statement took longer than a DDL lease. When the transaction was committed, the error occurred.
+- TiDB failed to connect to PD or TiKV while executing this DML statement. This caused TiDB to fail to load schema within a DDL lease or disconnect from PD due to the keepalive setting.
```

Comment:
```suggestion
- TiDB failed to connect to PD or TiKV while executing this DML statement. As a result, TiDB failed to load schema within a DDL lease or broke the connection containing the keepalive setting with PD.
```

Reply:
I'd prefer "disconnect" in this case as the connection was not actually "broken" by TiDB. It was broken because of the keep-alive setting

Reply:
What about "As a result, TiDB failed to load schema within a DDL lease or disconnected from PD due to the keepalive setting"? @lilin90

Reply:
@dcalvin Sounds good.